docs/core/tools/custom-templates.md

```diff
@@ -20,10 +20,10 @@ To follow a walkthrough and create a template, see the [Create a custom template
 
 ### .NET default templates
 
-When you install the [.NET SDK](https://dotnet.microsoft.com/download), you receive over a dozen built-in templates for creating projects and files, including console apps, class libraries, unit test projects, ASP.NET Core apps (including [Angular](https://angular.io/) and [React](https://reactjs.org/) projects), and configuration files. To list the built-in templates, run the `dotnet new` command with the `-l|--list` option:
+When you install the [.NET SDK](https://dotnet.microsoft.com/download), you receive over a dozen built-in templates for creating projects and files, including console apps, class libraries, unit test projects, ASP.NET Core apps (including [Angular](https://angular.io/) and [React](https://reactjs.org/) projects), and configuration files. To list the built-in templates, run the `dotnet new list` command:
 
 ```dotnetcli
-dotnet new --list
```

Comment:
@tdykstra Don't we need to show both syntaxes here since these docs are not versioned?

Reply:
We cover the old syntax in an include at the top of every article
![image](https://user-images.githubusercontent.com/1569635/199394347-282c9660-0265-43c8-bc5d-d6c2ac8925a2.png)